reference/channels.v3.yml

```
@@ -502,11 +502,118 @@ paths:
+                            },
+                            {
+                                "url": "https://store-abc.store.bcdev",
+                                "type": "canonical",
+                                "created_at": "2022-03-22T13:08:59Z",
+                                "updated_at": "2022-03-22T13:08:59Z"
+                            },
+                            {
+                                "url": "https://checkout.kittens.mybigcommerce.com",
+                                "type": "checkout",
+                                "created_at": "2022-03-23T10:21:11Z",
+                                "updated_at": "2022-03-29T13:44:22Z"
+                            }
+                        ],
+                        "is_checkout_url_customized": true
+                    }
+        '422':
+          description: Unprocessable Entity. Your checkout and storefront must be within the same main domain like `example.com` and `subdomain.example.com`
+          content:
+            application/json:
+              schema:
+                type: object
+                x-examples:
+                  example-1:
+                    status: 422
+                    title: 'Incorrect checkout url https://bc.kittykat.me. Your checkout and storefront must be within the same main domain like “main.com” and “subdomain.main.com”'
+                    type: 'https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes'
+                    errors: {}
+                properties:
+                  status:
+                    type: integer
+                    default: 422
+                  title:
+                    type: string
+                    example: 'Incorrect checkout url https://checkout.example.com. Your checkout and storefront must be within the same main domain like “main.com” and “subdomain.main.com'
+                  type:
+                    type: string
+                    default: 'https://developer.bigcommerce.com/api-docs/getting-started/api-status-codes'
+                  errors:
+                    type: object
+    delete:
+      summary: Delete a Site's Checkout URL
+      operationId: delete-checkout-url
+      description: 'Deletes a site''s checkout URL. After deletion, a shared checkout URL is used.'
+      tags:
+        - Channel Site Checkout Url
+      responses:
+        '200':
+          description: OK. `data` and `meta` are empty objects.
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  data:
+                    type: object
+                  meta:
+                    type: object
   '/stores/{store_hash}/v3/channels/{channel_id}/site':
     get:
       summary: Get a Channel Site
       description: |
-        Alias of GET /sites?channel_id=channel_id
+        Alias of `GET /sites?channel_id=channel_id`
 
         Returns site data for given channel.
```

```suggestion
        Returns site data for the specified channel.
```

I am not stoked on the `given channel` verbiage and prefer `the specified channel`. Can you find and replace?